packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -143,7 +162,7 @@ public function process(
                     $this->processPool->quitAll();
                 }
 
-                if ($jobs === []) {
+                if ($useSeparateProcessPerJob || $jobs === []) {
```

Comment:
when use separate process per job is enabled, always quit the actual process execution